src/main/java/org/jitsi/jicofo/xmpp/IqHandler.java

```diff
@@ -309,6 +309,8 @@ private IQ handleRayoIQ(RayoIqProvider.DialIq dialIq, int retryCount, @NotNull L
             return IQ.createErrorResponse(dialIq, XMPPError.getBuilder(XMPPError.Condition.not_allowed));
         }
 
+        final List<Jid> excludeList = exclude == null ? new ArrayList<>() : exclude;
```

Comment:
`exclude` is `@NotNull`

Reply:
Fixed it ... I was working on older version ...